src/containers/styled/index.js

```diff
@@ -1,30 +1,110 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { CardSegment, Colors, StyleUtils } from 'lattice-ui-kit';
 
-import { APP_CONTAINER_WIDTH } from '../../core/style/Sizes';
+import { ContentOuterWrapper, ContentWrapper } from '../../components/layout';
+import { APP_CONTAINER_WIDTH, HEIGHTS } from '../../core/style/Sizes';
 
-const { NEUTRAL } = Colors;
+const { NEUTRAL, PURPLE } = Colors;
 
 const { media } = StyleUtils;
 
-const MapWrapper = styled.div`
+const PADDING = 25;
+
+const DataRows = styled.div`
   display: flex;
-  min-height: 22rem;
-  height: 40vh;
-  max-width: ${APP_CONTAINER_WIDTH}px;
+  flex-direction: column;
+  ${(props) => (props.maxWidth ? css` max-width: ${props.maxWidth} !important; ` : '')}
+
+  span {
+    color: ${NEUTRAL.N600};
+    text-align: right;
+  }
+
+  a {
+    min-width: fit-content;
+    text-align: right;
+    ${(props) => (props.alignEnd ? css` align-self: flex-end; ` : '')}
+  }
 `;
 
-const SearchTitle = styled.h1`
+const TitleRow = styled.div`
+  align-items: center;
+  color: ${NEUTRAL.N600};
   display: flex;
-  font-size: 18px;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 20px 0;
+  width: 100%;
+
+  span {
+    color: ${NEUTRAL.N700};
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 17px;
```

Comment:
The `Typography` component is preferred as it's built to follow our design style guide.

Reply:
This will be taken care of in APPS-2546